tests/test_date_info.py

```diff
@@ -136,20 +138,20 @@ def test_prev_and_next_day(self, date: dt.date) -> None:
 
 
 UPCOMING_HOLIDAYS = [
-    ((2018, 8, 8), (2018, 9, 10), "rosh_hashana_i", "BOTH"),
-    ((2018, 9, 8), (2018, 9, 10), "rosh_hashana_i", "BOTH"),
-    ((2018, 9, 10), (2018, 9, 10), "rosh_hashana_i", "BOTH"),
-    ((2018, 9, 11), (2018, 9, 11), "rosh_hashana_ii", "BOTH"),
-    ((2018, 9, 12), (2018, 9, 19), "yom_kippur", "BOTH"),
-    ((2018, 9, 19), (2018, 9, 19), "yom_kippur", "BOTH"),
-    ((2018, 9, 20), (2018, 9, 24), "sukkot", "BOTH"),
-    ((2018, 9, 24), (2018, 9, 24), "sukkot", "BOTH"),
-    ((2018, 9, 25), (2018, 9, 25), "sukkot_ii", "DIASPORA"),
-    ((2018, 9, 25), (2018, 10, 1), "shmini_atzeret", "ISRAEL"),
-    ((2018, 9, 26), (2018, 10, 1), "shmini_atzeret", "BOTH"),
-    ((2018, 10, 2), (2018, 10, 2), "simchat_torah", "DIASPORA"),
-    ((2018, 10, 2), (2019, 4, 20), "pesach", "ISRAEL"),
-    ((2018, 10, 3), (2019, 4, 20), "pesach", "BOTH"),
+    ((2018, 8, 8), (2018, 9, 10), "rosh_hashana_i", "israel"),
```

Comment:
I think here too it is better to use str enum for the holidays across all the code